src/vmcs/src/ept_intel_x64.cpp

```diff
@@ -41,60 +41,66 @@ ept_intel_x64::ept_intel_x64(pointer epte)
 ept_entry_intel_x64
 ept_intel_x64::add_page(integer_pointer gpa, integer_pointer bits, integer_pointer end)
 {
-    auto &&index = ept::index(gpa, bits);
+    auto index = ept::index(gpa, bits);
+    auto view = gsl::make_span(m_ept, ept::num_entries);
```

Comment:
huh, not a bad idea. How about instead, a function that will return an entry object given an index